lightning-invoice/src/lib.rs

```diff
@@ -795,18 +795,18 @@ impl SignedRawInvoice {
 /// variant. If no element was found `None` gets returned.
 ///
 /// The following example would extract the first B.
-/// ```
-/// use Enum::*
+/// ``
+/// use Enum::*;
 ///
 /// enum Enum {
 /// 	A(u8),
 /// 	B(u16)
 /// }
 ///
-/// let elements = vec![A(1), A(2), B(3), A(4)]
+/// let elements = vec![A(1), A(2), B(3), A(4)];
 ///
-/// assert_eq!(find_extract!(elements.iter(), Enum::B(ref x), x), Some(3u16))
-/// ```
+/// assert_eq!(find_extract!(elements.iter(), Enum::B(ref x), x), Some(3u16));
+/// ``
```

Comment:
Does double ticking do anything? Thought that would just be the empty string formatted in markdown.